src/main/java/frc/robot/subsystems/Swerve/SwerveSubsystem.java

```diff
@@ -61,9 +65,10 @@ public SwerveModulePosition[] getPositions() {
         for (int i = 0; i < 4; i++) pos[i] = modules[i].getPosition();
         return pos;
     }
-	public void drive(ChassisSpeeds speeds) {
-		
-		
+	public void drive(double xVelocity, double yVelocity, double rotationalVelocity) {
+		ChassisSpeeds speeds =
+				ChassisSpeeds.fromFieldRelativeSpeeds(xVelocity, yVelocity, rotationalVelocity, POSE_ESTIMATOR.getEstimatedPose().getRotation());
+				speeds = ChassisSpeeds.discretize(speeds, 0.02);
```

Comment:
Please make 0.02 constant with appropiate name. What does this value mean?

Reply:
this value is hard to explain because it is a discrete mathematical function. I updated it to dtSeconds which is what the variable is formally known as in the equation